releaseNotes/AzureAppConfigurationProviderPython.md

```diff
@@ -2,6 +2,16 @@
 
 [Source code][source_code] | [Package (Pypi)][package] | [Samples][samples]
 
+## 1.1.0 - January 29, 2024
+
+### Features
+
+* Added support for dynamically refreshing configuration values with the new `refresh` method on `AzureAppConfigurationProvider`. This enables the runtime change of configuration values without restarting the application. A `WatchKey` can be set to trigger when a refresh can now occur.
+  * Added `refresh_on` parameter to the load method, selets which keys chaning should cause a refresh.
```

Comment:
typos, selects & changing? Also will that be keys or key for refresh_on?

Reply:
Fixed, I think. it should be key(s).